src/Umbraco.Web.BackOffice/Trees/ContentBlueprintTreeController.cs

```diff
@@ -31,54 +27,56 @@ namespace Umbraco.Web.Trees
     public class ContentBlueprintTreeController : TreeController
     {
         private readonly IMenuItemCollectionFactory _menuItemCollectionFactory;
+        private readonly IContentService _contentService;
+        private readonly IContentTypeService _contentTypeService;
+        private readonly IEntityService _entityService;
 
         public ContentBlueprintTreeController(
-            IGlobalSettings globalSettings,
-            IUmbracoContextAccessor umbracoContextAccessor,
-            ISqlContext sqlContext,
-            ServiceContext services,
-            AppCaches appCaches,
-            IProfilingLogger logger,
-            IRuntimeState runtimeState,
-            UmbracoMapper umbracoMapper,
-            IPublishedUrlProvider publishedUrlProvider,
-            IMenuItemCollectionFactory menuItemCollectionFactory)
-            : base(globalSettings, umbracoContextAccessor, sqlContext, services, appCaches, logger, runtimeState, umbracoMapper, publishedUrlProvider)
+            ILocalizedTextService localizedTextService,
+            UmbracoApiControllerTypeCollection umbracoApiControllerTypeCollection,
+            IMenuItemCollectionFactory menuItemCollectionFactory,
+            IContentService contentService,
+            IContentTypeService contentTypeService,
+            IEntityService entityService)
+            : base(localizedTextService, umbracoApiControllerTypeCollection)
         {
             _menuItemCollectionFactory = menuItemCollectionFactory;
+            _contentService = contentService;
```

Comment:
Do we ever do the assert is not null pattern on services injected into controllers?

Reply:
Sometimes :) Overall it is better to do it, but it is not very important